addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/handler/XiaomiDeviceBaseHandler.java

```diff
@@ -123,13 +123,17 @@ public String getItemId() {
     void parseCommand(String command, JsonObject data) {
         if (command.equals("report")) {
             parseReport(data);
-        } else if (command.equals("heartbeat") || command.equals("read_ack") || command.equals("write_ack")) {
+        } else if (command.equals("heartbeat")) {
+            parseHeartbeat(data);
+        } else if (command.equals("read_ack") || command.equals("write_ack")) {
```

Comment:
in XiaomiSensorBaseHandler and XiaomiActorGatewayHandler you seem to overwrite parseCommand to be able to parse read_ack and write_ack as parseHeartbeat. Maybe in here we should have them separate as well and call an abstract function parseReadAck or parseWriteAck. Then have XiaomiActorBaseHandler implement default functions for them. Or you can in here add them as just empty methods that are not abstract.
Does that make sense or is there a reason to better have it this way?